packages/asc-ui/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Close } from '@datapunt/asc-assets'
+import Tag, { Props as TagProps } from '../Tag/Tag'
+import FilterTagStyle from './FilterTagStyle'
+import Icon from '../Icon'
+
+export interface FilterTagProps extends TagProps {}
+
+const FilterTag: React.FC<FilterTagProps> = ({
+  filterTagOnClick,
+  children,
+  ...otherProps
+}) => {
+  const handleClick = (e: Event) => {
```

Comment:
I believe we look for mouse events and keyboard events specifically, look at thow this is done for other components like `Button`